e2e/test/scenarios/dashboard-filters/reproductions/8030-32444-reload-card-without-change.cy.spec.js

```diff
@@ -120,3 +171,9 @@ const interceptRequests = ({ dashboard_id, card1_id, card2_id }) => {
     `/api/dashboard/${dashboard_id}/dashcard/*/card/${card2_id}/query`,
   ).as("getCardQuery2");
 };
+
+function addFilterValue() {
+  filterWidget().click();
+  cy.findByPlaceholderText("Enter some text").type("5{enter}");
+  cy.button("Add filter").click();
+}
```

Comment:
WDYT about adding a `value` argument to `addFilterValue`? Seeing it called without any arguments made me think we're not actually filling any value

Reply:
I had the similar idea, but wasn't sure about it. fixed!